index.php

```diff
@@ -1,18 +1,18 @@
 <!DOCTYPE HTML>
-<!-- Author: Ashley Burnett -->
 
 <?php
 //Used for browser title and for META tags.
+include 'functions.php';
 $title = "Medieval Latin Lessons - The National Archives"
 ?>
 
 <!-- no-js class is needed for Modernizr -->
 <html lang="en-gb" class="no-js">
 <head>
     <title><?= $title ?></title>
-	<?php
-	include 'inc/head/head.php';
-	?>
+    <?php
+    include 'inc/head/head.php';
```

Comment:
We need to have a quick catch up to discuss all the includes just so you can get a better understanding of how we use them.